_components/pages/Docs.tsx

```diff
@@ -0,0 +1,424 @@
+      <div id="video-overlay"></div>
+      <div id="video">
+        <div id="videoWrapper">
+          <iframe
+            id="player"
+            width="853"
+            height="480"
+            src=""
+            frameBorder="0"
+            allowFullScreen={true}
+          ></iframe>
+        </div>
+      </div>
+      <section className="py-26 text-center">
+        <div className="col-xl-4 col-lg-6 mx-auto text-center">
+          <div className="d-flex flex-column-reverse w-100">
+            <h1 className="mb-18">XRP Ledger Developer Resources</h1>
+            <h6 className="eyebrow mb-3">Documentation</h6>
+          </div>
+        </div>
+      </section>
+
+      <section className="py-10 px-20 text-left">
+        <div className="mx-auto mb-10">
+          <h4>Getting Started with XRP Ledger</h4>
+        </div>
+        <div className="card-grid card-grid-2xN">
+          <div className="col card" id="quick-start">
+            <h5 className="mt-7"> Quickstart to XRP Ledger </h5>
+            <p className="mb-8 mt-4">
+              An introduction to fundamental aspects of the XRP Ledger.
+            </p>
+            <div className="dg-lg-block mb-5">
+              <a
+                className="btn btn-primary btn-arrow get-started-button"
+                href="https://xrpl.org/xrpl-quickstart.html"
+              >
+                Get Started
+              </a>
+            </div>
+            <img
+              src={prefix + "/img/docs/docs-quick-start.svg"}
+              alt="quick-start"
+              id="quick-start-img"
+            />
+          </div>
+          <div className="col">
+            <div className="card-grid card-grid-2xN">
+              {getStartedVideos.map((video) => (
+                <VideoCard props={video} />
+              ))}
+            </div>
+            <a
+              className="btn btn-primary btn-arrow-out mt-10"
+              href="https://www.youtube.com/playlist?list=PLJQ55Tj1hIVZtJ_JdTvSum2qMTsedWkNi"
+            >
+              Watch Full Series
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <section className="py-26 px-20 text-left">
+        <div className="card-grid card-grid-2xN">
+          <div className="col">
+            <h6>Explore SDKs</h6>
+            <h4>Interact with the XRP Ledger in a language of your choice</h4>
+            <div className="card-grid card-grid-2xN mt-10">
+              <div className="col langs">
+                <img
+                  src={prefix + "/img/docs/javascript.svg"}
+                  className="circled-logo"
+                ></img>
+                <h5>
+                  <a
+                    className="btn-arrow-purple"
+                    target="_blank"
+                    href="https://github.com/XRPLF/xrpl.js"
+                  >
+                    Javascript
+                  </a>
+                </h5>
+              </div>
```

Comment:
I think the whole block—especially the icon—should be clickable, not just the text. Here's one way you could do that (I think it'll need CSS fixes to make the hover color look right for the text though):

```suggestion
              <a className="col langs" target="_blank"
                    href="https://github.com/XRPLF/xrpl.js">
                <img
                  src={prefix + "/img/docs/javascript.svg"}
                  className="circled-logo"
                ></img>
                <h5 className="btn-arrow-purple">
                    Javascript
                </h5>
              </a>
```